### DISCUSSION TITLE
Colab example notebook

### DISCUSSION THREAD

**Discussion by bth5** [open-edge-platform/anomalib#157] [Show and tell]
it would really help if there would be a proof of concept working colab available to test the functionality of the library. I'm struggling to get up and running. I think it would be a good stepping stone to use these amazing resources.

**Comment by bth5** [comment]
I've made a test notebook now that runs the basic bottle dataset from MVTec using the PADIM model and test the output model on two samples. 

There are not much to it, actually - but i hope it can help others who wants to test the library. 

Let me know if something needs changing of you are having trouble. 

[Link to notebook](https://colab.research.google.com/drive/1K4a4z2iZGBNhWdmt9Aqdld7kTAxBfAmi?usp=sharing )

**Reply by ashwinvaidya17** [reply]
Thanks for sharing!
